### ISSUE TITLE
Too many annotations are read-only

### ISSUE THREAD

**Issue by vincent99** [rancher/rancher#23849]
Issue rancher/rancher#22331 and PR [#3296](https://github.com/rancher/ui/pull/3296) made various labels and annotations read-only in the UI.  But it is is too aggressive and is making things read-only that should be user-editable.

The primary offender is that Annotations looks for "contains `kubernetes.io`" instead of "starts with `kubernetes.io/` like the Labels section.  So that blocks `nginx-ingress.kubernetes.io/*` and similar.  But someone should probably go through what else is blocked in both sections and see if it always makes sense.

**Comment by westlywright** [comment]
Available in `latest-2.3`.

**Comment by jiaqiluo** [comment]
The bug fix is validated in v2.3-head 5187fce31 with UI index `latest-2.3`

If the key of the annotation starts with one of the following prefixes, this annotation will be un-editable in the UI; Otherwise, it will be editable. 
```
  'kubernetes.io/',
  'beta.kubernetes.io/',
  'node.alpha.kubernetes.io/',
  'volumes.kubernetes.io/',
```

for example, the annotation  `kubernetes.io/test = abc` is not editable., but `nginx-ingress.kubernetes.io/test = abc` is editable
 
 
<img width="689" alt="Screen Shot 2019-11-11 at 1 14 47 PM" src="https://user-images.githubusercontent.com/6218999/68618065-c5910b00-0485-11ea-9c13-6fbeb608a718.png">
